features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/application-authentication.xml

```diff
@@ -174,6 +174,7 @@
             <Parameter name="windowSize">3</Parameter>
             <Parameter name="authenticationMandatory">true</Parameter>
             <Parameter name="enrolUserInAuthenticationFlow">true</Parameter>
+            <Parameter name="useEventHandlerBasedEmailSender">false</Parameter>
```

Comment:
Why is this false? should be true going forward for GAs.

Reply:
Also, is this done for template file?

Reply:
set to true by default both in the template file and application-authentication.xml